.env.example

```diff
@@ -0,0 +1,2 @@
+PORT=8080
```

Comment:
IMO, we should not push `.env` file to the public as we may have some sensitive information in the `.env` file.
What do you guys think?

Reply:
Yes, @gutakk that's why I use `.env.example` instead of `.env` just in case we need to add any sensitive variables in the future 🤔  what do you think @junan?

Reply:
The reason I said is that in the dockerfile we are copying the content to of `.env.example` to `.env`. In this sense, they are kind of same file.
 
> ADD .env.example ./.env

We usually use `.env.example` only for showing an example, but here we are using it in real case(`ADD .env.example ./.env`).   May be we can rename it to some other name like `.env.docker-example` or any better name 🤔  . @carryall @gutakk What do you think guys?

Reply:
@junan IMO I still think we can go with the current file name since we are going to use this file as the example for the `.env` file later and I think we shouldn't change it to something related to docker because the variables in the file also used on `app.conf` 🤔